salary/src/main/java/cz/fi/muni/pv217/employee/management/service/CoreService.java

```diff
@@ -4,20 +4,12 @@
 import cz.fi.muni.pv217.employee.management.pojo.Workday;
 import org.eclipse.microprofile.rest.client.inject.RegisterRestClient;
 
-import javax.enterprise.context.ApplicationScoped;
-import javax.inject.Singleton;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
-import javax.ws.rs.QueryParam;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
-import java.time.LocalDate;
 import java.util.List;
 
 @Path("/api")
 @RegisterRestClient(configKey="core-api")
```

Comment:
Dont you want to also change hits key to core-service?

Reply:
Yes I will change it also here.